src/plugins/discover/public/application/main/components/layout/use_discover_histogram.ts

```diff
@@ -184,11 +182,15 @@ export const useDiscoverHistogram = ({
           return;
         }
 
-        // Sync the totalHits$ observable with the unified histogram state
-        savedSearchData$.totalHits$.next({
-          fetchStatus: status.toString() as FetchStatus,
-          result,
-        });
+        const fetchStatus = status.toString() as FetchStatus;
+
+        // Do not sync the loading state since it's already handled by the saved search
+        if (fetchStatus !== FetchStatus.LOADING) {
+          savedSearchHits$.next({
```

Comment:
I think there was a good reason why the histogram was sending the `loading` status all this time. But I don't remember all the case about that. Do you mind if we wait for @davismcphee review?

Reply:
yes, let's wait, from what I've seen, it doesn't hurt keeping it, but on the other hand, I tought it's redundant